src/encoded/locust/main.py

```diff
@@ -0,0 +1,40 @@
+import os
+import argparse
+import subprocess
+
+def main():
+    """
+    Entry point for locust load testing. Run with -h to see arguments
+    """
+    parser = argparse.ArgumentParser(
+        description='Locust Load Testing',
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument('config', help='path to config.json')
+    parser.add_argument('key', help='path to <env>.json')
+    parser.add_argument('--time', help='time to run test for, default 1m. Format: 10s, 5m, 1h, 1h30m etc.')
+    parser.add_argument('--clients', help='number of clients, default 10')
```

Comment:
I have a slight preference for calling the arguments `-nclients` or some such thing (maybe even with a short name of `-n`). The reason is that I vaguely expect an argument named `--clients` to have me list what the clients are, and so to feel like I'll have to go look some syntax up. Something that more strongly suggests a number feels better. I admit that I usually also prefer ordinary English words and can't defend `nclients` under that theory, but somehow it feels good to me. I'll leave all this to your best judgment. I just wanted to mention the issue.

(And see other comments for suggestions about defaults.)

Reply:
I like ``nclients`` as well and use the convention oftentimes myself. Changed.